src/js/components/List/List.js

```diff
@@ -357,13 +371,17 @@ const List = React.forwardRef(
                       // accessibility guidelines that focus remains on `ul`
                       listRef.current.focus();
                     },
-                    onMouseOver: () => setActive(index),
+                    onMouseOver: () => {
+                      setLastActive(undefined);
```

Comment:
I don't think we need to `setLastActive` here.

Reply:
After removal, functionality works as intended